tmt/base.py

```diff
@@ -1018,7 +1013,11 @@ def _lint_keys(self, additional_keys: list[str]) -> list[str]:
 
         known_keys.extend(additional_keys)
 
-        return [key for key in self.node.get() if key not in known_keys]
+        return [
+            key
+            for key in self.node.get()
+            if key not in known_keys and not key.startswith('extra-')
```

Comment:
I'd put the `'extra-'` string somewhere on the module level, e.g. below `EXTRA_TEST_KEYS`, something like `EXTRA_KEYS_PREFIX`. We should avoid magic values in the code.